client/src/modules/prices/modal.js

```diff
@@ -14,10 +14,27 @@ function PriceListModalController( $uibModalInstance, Inventory, util, Notify) {
   vm.action = null;
   vm.isValidItem = isValidItem;
 
-  // load Inventory
+  // Here we load the All items from the list
+  const cache = new AppCache('selecedItems');
```

Comment:
We cannot use `const` or `let` or any ES2015+ features on the client.  It will break our production builds.  These should be `var`s until we start transpiling to ES5.